contrib/libhdfs3-cmake/CMakeLists.txt

```diff
@@ -99,20 +99,26 @@ set(SRCS
     "${HDFS3_SOURCE_DIR}/common/Hash.cpp"
     "${HDFS3_SOURCE_DIR}/common/SWCrc32c.cpp"
     "${HDFS3_SOURCE_DIR}/common/Thread.cpp"
+    "${HDFS3_SOURCE_DIR}/common/IntelAsmCrc32c.cpp"
+    # ""
     ${PROTO_SOURCES}
 )
 
 # old kernels (< 3.17) doesn't have SYS_getrandom. Always use POSIX implementation to have better compatibility
 set_source_files_properties("${HDFS3_SOURCE_DIR}/rpc/RpcClient.cpp" PROPERTIES COMPILE_FLAGS "-DBOOST_UUID_RANDOM_PROVIDER_FORCE_POSIX=1")
 
+find_program(YASM_EXE NAMES yasm)
```

Comment:
"YASM_PATH" instead of "YASM" would be more consistent with the naming in ClickHouse.

Reply:
Would be nice to provide a COMMENT to add_custom_command, e.g. `COMMENT "Translating Intel CRC assembly to object code"`.

Reply:
Good advice, thanks !